src/config/constants/farms.ts

```diff
@@ -12,6 +12,2441 @@ const farms: FarmConfig[] = [
+  },
+  {
+    pid: 280,
+    lpSymbol: 'TEN-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x1B415C3ec8095AfBF9d78882b3a6263c4ad141B5',
+    },
+    token: tokens.ten,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 279,
+    lpSymbol: 'bALBT-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x24EB18bA412701f278B172ef96697c4622b19da6',
+    },
+    token: tokens.balbt,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 278,
+    lpSymbol: 'REEF-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xd63b5CecB1f40d626307B92706Df357709D05827',
+    },
+    token: tokens.reef,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 277,
+    lpSymbol: 'Ditto-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x8645148dE4E339964bA480AE3478653b5bc6E211',
+    },
+    token: tokens.ditto,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 276,
+    lpSymbol: 'VAI-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0x133ee93FE93320e1182923E1a640912eDE17C90C',
+    },
+    token: tokens.vai,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 275,
+    lpSymbol: 'BLK-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x356Dd24BfF8e23BdE0430f00ad0C290E33438bD7',
+    },
+    token: tokens.blink,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 274,
+    lpSymbol: 'UNFI-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x44EA47F2765fd5D26b7eF0222736AD6FD6f61950',
+    },
+    token: tokens.unfi,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 273,
+    lpSymbol: 'HARD-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x73566ca86248bD12F0979793e4671e99a40299A7',
+    },
+    token: tokens.hard,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 272,
+    lpSymbol: 'CTK-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x460b4193Ec4C1a17372Aa5FDcd44c520ba658646',
+    },
+    token: tokens.ctk,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 271,
+    lpSymbol: 'SXP-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xD8E2F8b6Db204c405543953Ef6359912FE3A88d6',
+    },
+    token: tokens.sxp,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 270,
+    lpSymbol: 'INJ-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x1BdCebcA3b93af70b58C41272AEa2231754B23ca',
+    },
+    token: tokens.inj,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 269,
+    lpSymbol: 'FIL-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xD9bCcbbbDFd9d67BEb5d2273102CE0762421D1e3',
+    },
+    token: tokens.fil,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 268,
+    lpSymbol: 'UNI-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x014608E87AF97a054C9a49f81E1473076D51d9a3',
+    },
+    token: tokens.uni,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 267,
+    lpSymbol: 'YFI-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xCE383277847f8217392eeA98C5a8B4a7D27811b0',
+    },
+    token: tokens.yfi,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 267,
+    lpSymbol: 'YFII-BNB LP',
```

Comment:
Is this a duplicate? Or deliberately have two different LPs with the same pid/address?